game/hud/src/components/HUD/index.tsx

```diff
@@ -83,6 +84,11 @@ const SkillBarContainer = styled('div')`
   pointer-events: none;
 `;
 
+interface HUDWidget {
```

Comment:
We should probably do something like
`interface HUDWidget<T> {
  widget: Widget<T>;
  name: string;
}`
instead of passing in `any` to widget. I know this interface isn't used in many places and doing this wouldn't have much of an effect with this interface, but its just good to keep standard.

Reply:
I've implemented all other requests and will work on this one. Might end up needing help, but believe I have a good idea to fix the million any's that I inherited from the old HUD editor. :)

Reply:
Hehe I just saw discord chat, I didn't know the use of the interface when I was going through the review so that was my bad.

What you did here before and after commit is fine (you're right they pretty much are the same thing just diff syntax :P), and an `any` here doesn't have that much of an impact.